play/benchmarker/public/javascripts/hello.js

```diff
@@ -0,0 +1,3 @@
+if (window.console) {
+  console.log("Welcome to your Play application's JavaScript!");
```

Comment:
The other applications don't serve this file, so we should probably remove for a more fair comparison.